docs/source/index.rst

```diff
@@ -20,4 +20,5 @@ Contents:
    VUDRM-token.md
```

Comment:
Could you put this list into alphabetical order?

Reply:
Dont you think it should ordered by relevance?, geo-location is far less important than Token.
You never see Contact us and FAQ sections at the top of a menu

Reply:
Thats a not a bad point, @fschurer is probably the person to decide?

Reply:
Sure yea just did when then i thought about it, I dont really mind that much tbh. Either way, reordering the menu is not what this PR is about, is it?

Reply:
I guess not, but where this documentation come in the list is directly affected by the out come of that discussion.